ethereum/statetransition/src/testFixtures/java/tech/pegasys/teku/statetransition/datacolumns/DasCustodyStand.java

```diff
@@ -216,11 +213,6 @@ public Builder withSpec(final Spec spec) {
       return this;
     }
 
```

Comment:
There are many other not used `with..`. It's just a builder. Not sure we need to remove smth